src/lib/components/ExpertDataCard.tsx

```diff
@@ -8,17 +8,18 @@ import { DIRECTION_PROPERTIES } from './PostPreview/direction-properties';
 import { useStyles } from '../styles/ExpertDataCard.styles';
 
 export interface IExpertDataCardProps {
-  expert: IExpert;
+  expert: IExpert | null;
```

Comment:
`ExpertDataCard` should not have `expert` as optional.
there's no point to render full info for an expert if there's no expert :)

instead, you can validate the expert on the higher level and render the `ExpertDataCard` only if `expert` is defined.

Reply:
fixed